mitodl/ocw-studio
# course_feature_tags->learning_resource_types

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-hugo-projects/issues/58

#### What's this PR do?
This updates localdev/configs/ocw-course-site-config.yml to match the changes in https://github.com/mitodl/ocw-hugo-projects/issues/58

#### How should this be manually tested?
Run `docker-compose run web python manage.py override_site_config`
Create a site with the ocw-course starter
Verify that you can create a resource for your site and set it's Learning Resource Type fields
